docs/tools/glossary.md

```diff
@@ -124,7 +124,7 @@ It is the first DPOS blockchain equipped with IBC. Its goal is to refactor busin
 
 ### Bytecode canonical
```

Comment:
Capitalization in Titles

Reply:
It tought I made this but it looks i didn't